modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -0,0 +1,79 @@
+
+== Share With an External Organization
+
+Exchange administrators and asset administrators can grant viewer access for an asset to everyone in a chosen external organization.
+
+The organization granting viewer access is the granting organization, and the organization that receives access is the receiving organization.
+
+. The receiving organization adds the granting organization to its "External Access" list in Anypoint Access Management, as described in xref:access-management::external-organization-access.adoc[External Organization Access].
++
+Assets can only be shared with another organization if that organization has enabled receiving viewer access to assets from external organizations.
++
+. The granting organization adds the receiving organization to its "External Access" list in Anypoint Access Management, as described in xref:access-management::external-organization-access.adoc[External Organization Access].
+. Login to your Exchange administrator or asset administrator account in the granting organization.
+. Click an asset to show its details.
+. Click *Share*.
+. Begin typing the name of the external organization you want to share this asset with.
+. Select the organization in the list.
+. Repeat steps 6 and 7 to share with additional organizations.
+. Click *Save*.
+
+When an asset is shared with an organization, email notification is sent to the organization administrators.
+
+You can also click *Share* to change who you are sharing the asset with or to stop sharing with one or more organizations.
+
+Only internal users in the same organization as the asset can request access to an API or gain the contributor or administrator roles.
+
+=== Using Assets Shared with External Organizations
+
+If a connector is shared with an external organization, then in Studio or Flow Designer a user in the external organization can search for the connector, configure it, and use it in a Mule application.
+
+If an example or template is shared with an external organization, then in Studio or Flow Designer a user in the external organization can implement a Mule application based on the shared template.
+
+If a REST, SOAP, or HTTP API is created in Exchange and shared with an external organization, then API Manager cannot manage the API in the external organization. API Manager search results show only APIs from the external organization user's business group. In Studio a user in the external organization can use the API in the flow as a source for scaffolding an application.
+
+If a RAML fragment is created in Design Center and shared with an external organization, then in API Designer a user in the external organization can import the fragment and add it as a dependency in the new API specification. If the fragment is unshared, then the API specification in the external organization shows the same error that is shown if the fragment is deleted, "File not found."
```

Comment:
DITTO previous sentence restructuring suggestions for the external user using <product> to <do this>.

Reply:
Why are we documenting what happens if something is not shared, when that is the focus of this topic set?

Reply:
Rewritten for clarity.

By "unshared" I meant that the granting organization shares it and then later stops sharing it. Rewrote to clarify that.